ENH: add `_new_tests` to allow instruments to temporarily ignore tests requiring concat (#1166)
pysat/pysat: docs/new_instrument.rst

   _test_dates = {'': {'Level_1': dt.datetime(2020, 1, 1),
                       'Level_2': dt.datetime(2020, 1, 1)}}
   _password_req = {'': {'Level_1': True}}

Updates to Instrument Suite Tests
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> Sometimes new standard tests are added to pysat that update compliance
> requirements throughout the ecosystem. For example, pysat 3.2.0 adds new tests

Reviewer: ```suggestion
Sometimes new standard tests are added to pysat that ensure all data handling features
work as expected throughout the ecosystem. For example, pysat 3.2.0 adds new tests
```

for loading multiple days at a time or using a data pad. When these tests
require significant updates, an additional flag may be used to suppress these

> tests temporarily for specific instruments while updates are made throughout
> the ecosystem. These new tests are run by default unless specified using the

Reviewer: ```suggestion
tests temporarily for specific instruments while updates are made to that
instrument. These new tests are run by default unless specified using the
```
